google-cloud-pubsub/test/google/cloud/pubsub/subscriber/timed_unary_buffer_test.rb

```diff
@@ -78,6 +78,9 @@ def stub.acknowledge subscription:, ack_ids:
     stub = StreamingPullStub.new response_groups
     called = false  
     def stub.modify_ack_deadline subscription:, ack_ids:, ack_deadline_seconds:
+      if @modify_ack_deadline_requests.count == 0
+        return  @modify_ack_deadline_requests << [subscription, ack_ids.sort, ack_deadline_seconds]
```

Comment:
nit: extra space. Same in 3 other cases below.

Reply:
Fixed 👍